src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -199,6 +193,13 @@ public void readFromNBT(@NotNull final NBTTagCompound compound)
         {
             setJob(AbstractJob.createFromNBT(this, compound.getCompoundTag("job")));
         }
+
+        if (compound.hasKey("inventory"))
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "inventory" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1192)